lua/compe/completion.lua

```diff
@@ -95,18 +95,20 @@ end
 
 --- select
 Completion.select = function(index)
-  if index == -1 then
-    return
-  end
+  if Config.get().show_documentation_as_float then
+    if index == -1 then
```

Comment:
This check should be moved to L107.

We must set `_selected_item` even if documentation disabled.